### PR TITLE
Improve efficiency of assigning badges as an Admin 

### PR DESCRIPTION
**Pull request by aaronleechan**
* Add Validate Badges in badgeManagement.js to check the user input firstName and lastName.
* If the user didn't fill up first and last name, show the error modal message box.
* else the assign badge pop up will appear.
* Comment Submit Button

### CODE REVIEW

File: src/components/Badge/AssignBadge.jsx [hunk 1]
```diff
@@ -74,13 +73,25 @@ const AssignBadge = (props) => {
...
+        props.validateBadges(firstName,lastName)
+      }
+    }
+  };
 
-  const clickSubmit = () => {
-    props.assignBadges(props.firstName, props.lastName, props.selectedBadges);
-    props.clearNameAndSelected();
-  }
+  // const clickSubmit = () => {
```
Comment (ireneclaregs) [inline src/components/Badge/AssignBadge.jsx:91]: Can we remove commented code?

**Comment by aaronleechan** [comment]
@ireneclaregs  Please take a look. I updated it.

File: src/actions/badgeManagement.js [hunk 2]
```diff
@@ -61,19 +61,21 @@ export const getMessage = (message, color) => ({
...
 
-
-
-export const assignBadges = (firstName, lastName, selectedBadges) => {
-
-  return async (dispatch) => {
-
+export const validateBadges = (firstName,lastName) => {
+  return async (dispatch) =>{
     if (firstName.length === 0 || lastName.length === 0) {
```
[the remark this answers is not in the document]
Comment (aaronleechan) [reply src/actions/badgeManagement.js]: Done.

File: src/components/Badge/AssignBadge.jsx (hunk 1 above)
Comment (aaronleechan) [reply src/components/Badge/AssignBadge.jsx]: Remove is Done.

**Pull request merged**